sbom/README.md

```diff
@@ -0,0 +1,13 @@
+# `sbomb` Extension for Phylum
```

Comment:
💣 

```suggestion
# `sbom` Extension for Phylum
```

Reply:
Or maybe this was intentional as a play on words? This typo is used many times in this file.

If it's not a typo, then the folder name should change.

Reply:
nope, I typed it up quickly 💣

Reply:
In that case, I've got a few more comments to add 😄

Reply:
haha, comment away!